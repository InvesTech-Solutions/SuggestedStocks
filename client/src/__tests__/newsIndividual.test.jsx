import React from 'react';
import NewsIndividual from '../components/newsIndividual.jsx';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

var data = {
    source: 'Yahoo Finance',
    title: 'Stocks, Stocks, and more Stocks',
    description: 'Investing.com-Technology Stocks',
    viewership: 466,
    url: '#',
    img: 'http://lorempixel.com/400/200',
    time: '7h'
}



test('renders', () => {
    const wrapper = shallow(<NewsIndividual data={data} />);

    expect(toJson(wrapper)).toMatchSnapshot();
})