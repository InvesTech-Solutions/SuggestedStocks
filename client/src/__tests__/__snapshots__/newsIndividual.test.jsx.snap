// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  id="mainNews"
>
  <img
    src="http://lorempixel.com/400/200"
  />
  <div
    id="newsInfoContainer"
  >
    <div
      id="Newsheaders"
    >
      <div
        id="articleTime"
      >
        <h5
          id="dataSource"
        >
           
          Yahoo Finance
           
        </h5>
        <h6
          id="time"
        >
           
          7h
           
        </h6>
      </div>
      <h4
        id="dataTitle"
      >
         
        Stocks, Stocks, and more Stocks
         
      </h4>
      <p
        id="dataDescription"
      >
         
        Investing.com-Technology Stocks
         
      </p>
    </div>
    <div
      id="views"
    >
      <svg
        height="11"
        id="theEye"
        title="494 views"
        viewBox="0 0 14 11"
        width="14"
      >
        <path
          d="M7,10.8888889 C3.13400675,10.8888889 0,7.04766168 0,5.44444444 C9.81687425e-17,3.84122721 3.13400675,0 7,0 C10.8659932,0 14,3.84312609 14,5.44444444 C14,7.0457628 10.8659932,10.8888889 7,10.8888889 Z M7,7.77777778 C8.28866442,7.77777778 9.33333333,6.73310886 9.33333333,5.44444444 C9.33333333,4.15578003 8.28866442,3.11111111 7,3.11111111 C5.71133558,3.11111111 4.66666667,4.15578003 4.66666667,5.44444444 C4.66666667,6.73310886 5.71133558,7.77777778 7,7.77777778 Z"
          fillRule="evenodd"
        />
      </svg>
      <p
        className="numberViews"
      >
         
        466
      </p>
    </div>
  </div>
</div>
`;
